packages/SwingSet/src/kernel/kernel.js

```diff
@@ -263,7 +263,7 @@ export default function buildKernel(
 
       // ISSUE: terminate stuff in its own crank like creation?
       // eslint-disable-next-line no-use-before-define
-      vatWarehouse.vatWasTerminated(vatID);
+      void vatWarehouse.vatWasTerminated(vatID);
```

Comment:
that seems ok, the worker might still be running (we've started `manager.shutdown()` but haven't `await`ed it), but everything else about the worker is gone, so I don't think it will hurt to let that happen in the background